src/app/Shared/ClearSelectorButton.component.tsx

```diff
@@ -20,6 +25,15 @@ const iconButtonStyle = css`
     color: ${primaryColor};
 `;
 
+const LightTooltip = withStyles(() => ({
+    tooltip: {
+        backgroundColor,
+        color: onBackgroundColor,
+        fontSize: 9,

```

Comment:
no unit specified. Consider using relative units

Reply:
Material UI itself specifies the font size without unit. If I use a relative unit, the appearance of the tooltip does not run smoothly.

Reply:
I add "px" as unit

Reply:
px is the default if no unit is specified i think. Perhaps Material UI does some internal scaling on the font size so that relative units doesnt work? You should check what happens if you scale the font size in the browser.